fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,53 @@ then
+
+# Defaults
+DoSliceTimeCorrection=${DoSliceTimeCorrection:-FALSE}                   # WARNING: This legacy option of slice timing correction is performed before motion correction 
+                                                                        # (as is typically done in legacy-style brain imaging) and thus assumes that the brain is motionless. 
+                                                                        # Errors in temporal interpolation will occur in the presence of head motion and may also disrupt 
+                                                                        # data quality measures as shown in Power et al 2017 PLOS One "Temporal interpolation alters motion in fMRI
+                                                                        # scans: Magnitudes and consequences for artifact detection." Slice timing correction and motion correction 
+                                                                        # would ideally be performed simultaneously; however, this is not currently supported by any major software 
+                                                                        # tool. HCP-Style fast TR fMRI data acquisitions (TR<=1s) avoid the need for slice timing correction, 
+                                                                        # provide major advantages for fMRI denoising, and are recommended. 
+                                                                        # No slice timing correction is done by default 
+SliceCorrectionInterleaved=${SliceCorrectionInterleaved:-TRUE}          # Interleaved slice timing is TRUE by default 
+
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+ProcessingMode=`opts_GetOpt1 "--processing-mode" $@`
+ProcessingMode=`opts_DefaultOpt $ProcessingMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+ComplianceWarn=""
+
+# -- Slice timing correction
+
+if [ "${DoSliceTimeCorrection}" = 'TRUE' ]; then
+  ComplianceMsg+=" --doslicetime=TRUE"
+  Compliance="LegacyStyleData"
+  log_Warn "WARNING: This legacy option of slice timing correction is performed before motion correction (as is typically done in legacy-style brain imaging) and thus assumes that the brain is motionless. Errors in temporal interpolation will occur in the presence of head motion and may also disrupt data quality measures as shown in Power et al 2017 PLOS One 'Temporal interpolation alters motion in fMRI scans: Magnitudes and consequences for artifact detection.' Slice timing correction and motion correction would ideally be performed simultaneously; however, this is not currently supported by any major software tool. HCP-Style fast TR fMRI data acquisitions (TR<=1s) avoid the need for slice timing correction, provide major advantages for fMRI denoising, and are recommended."
```

Comment:
For the same reasons as articulated previously, referring to something as a "legacy option" is highly ambiguous.  I would instead say: "This LegacyStyleData option of slice timing correction ..."  
(And make some change to the commented section).

Reply:
ok